.github/workflows/ci.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: gem install mdl
 
       - name: Lint markdown
-        run: mdl --ignore-front-matter --style .mdl_style.rb ./
+        run: mdl ./
```

Comment:
Does this now ignore the style?

In any case, if we are using pre-commit then the CI jobs should be `pre-commit run -a markdown-lint`, `pre-commit run -a hugo` (plus any extra jobs). Otherwise the mismatch between pre-commit and CI will just cause confusion.

Reply:
> Does this now ignore the style?

Nope, it still respects it! `mdl` will pick up `.mdlrc` from the repository root, which in turn specifies the command-line options that were removed.

> In any case, if we are using pre-commit then the CI jobs should be [...]

That sounds brilliant, let's do that.